style.css

```diff
@@ -0,0 +1,115 @@
+/* adding dimensions, color, shappe, font to My Logo */
+.my-logo {
+  height: 16px;
+  width: 68px;
+  left: 7px;
+  color: #6070ff;
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+}
+
+/* positioning all the elements in the header element inline */
+.header-nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+/* positioning my social media icons */
+figure {
+  width: 160px;
+  display: flex;
+  justify-content: space-between;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+/* positioning the whole body and placing a background color */
+body {
+  position: relative;
+  width: 375px;
+  height: 5410px;
+  background: #f7f7f9;
+}
+
+/* Working on header position */
+header {
+  display: flex;
+  flex-direction: column;
+  padding: 114px 24px;
```

Comment:
- [ ] Kindly revise the padding used here as it is resulting in your header section have large amounts of padding and differ from the figma design as seen below.

- figma desig
![Screenshot from 2021-10-21 08-01-14](https://user-images.githubusercontent.com/61377322/138214961-7de1e1c4-9396-428f-b280-5fa66005127c.png)
n
![Screenshot from 2021-10-21 08-05-57](https://user-images.githubusercontent.com/61377322/138214973-6dfbbc4a-4046-4bfb-8a5a-0b1b537cf8bc.png)

Reply:
Thank you for addressing this concern. I have matched the padding to the figma design